relay-server/src/services/store.rs

```diff
@@ -673,6 +689,36 @@ impl StoreService {
         self.produce(KafkaTopic::Attachments, message)
     }
 
+    fn produce_user_report_v2(
+        &self,
+        event_id: EventId,
+        project_id: ProjectId,
+        organization_id: u64,
+        start_time: Instant,
+        item: &Item,
+        remote_addr: Option<String>,
+    ) -> Result<(), StoreError> {
+        // check rollout rate option (effectively a FF) to determine whether to produce to new infra
+        let global_config = self.global_config.current();
+        let feedback_ingest_topic_rollout_rate =
+            global_config.options.feedback_ingest_topic_rollout_rate;
+        let topic = if is_rolled_out(organization_id, feedback_ingest_topic_rollout_rate) {
+            KafkaTopic::Feedback
+        } else {
+            KafkaTopic::Events
+        };
```

Comment:
Out of scope for this PR, but should we hard-code this to `Feedback` soon? Or are some installations not ready for that yet? (self-hosted, devenv, ...)

Reply:
Yes this has to do with ops deployment of the feedback topic -- right now, its only in s4s and not other regions. This check and rollout rate will eventually be removed too